apps/clock/onring.html

```diff
@@ -18,7 +18,8 @@
     <script defer src="/shared/js/l10n.js"></script>
     <script defer src="/shared/js/date_time_helper.js"></script>
     <script defer src="/shared/js/moz_intl.js"></script>
-    <script defer src="/shared/js/intl_helper.js"></script>
+    <!-- We need IntlHelper synchronously for onring.js -->
+    <script src="/shared/js/intl_helper.js"></script>
```

Comment:
no, because the only thing we do synchronously is defining IntlHelper formatters.